applications/vessel-history/src/features/profile/components/activity/ActivityVoyage.tsx

```diff
@@ -22,35 +22,37 @@ const ActivityVoyage: React.FC<EventProps> = ({
 
   const voyageLabel = useMemo(() => {
     const parts: string[] = []
+    const { from, to, start, end, eventsQuantity } = event
 
-    if (event.from && event.to) {
+    if (from && to) {
       parts.push(
-        `${t('common.from', 'from')} ${formatI18nDate(event.start ?? 0, {
-          format: DateTime.DATETIME_FULL,
+        `${t('common.from', 'from')} ${formatI18nDate(start ?? 0, {
+          format: DateTime.DATE_FULL,
         })}`
       )
       parts.push(
-        `${t('common.to', 'to')} ${formatI18nDate(event.end ?? 0, {
-          format: DateTime.DATETIME_FULL,
+        `${t('common.to', 'to')} ${formatI18nDate(end ?? 0, {
+          format: DateTime.DATE_FULL,
         })}`
       )
+      parts.push(`(${eventsQuantity} ${t('common.events', 'Events')})`)
     } else if (!event.to) {
       parts.push(t('event.currentVoyage' as any, 'Ongoing Voyage') as string)
       parts.push(
-        `${t('common.from', 'from')} ${formatI18nDate(event.start ?? 0, {
-          format: DateTime.DATETIME_FULL,
+        `${t('common.from', 'from')} ${formatI18nDate(start ?? 0, {
+          format: DateTime.DATE_FULL,
         })}`
       )
     } else {
       parts.push(
-        `${t('common.to', 'to')} ${formatI18nDate(event.end ?? 0, {
-          format: DateTime.DATETIME_FULL,
+        `${t('common.to', 'to')} ${formatI18nDate(end ?? 0, {
+          format: DateTime.DATE_FULL,
```

Comment:
So we will not display the time anymore in voyages?

Reply:
Yes, I don't think they're needed. And removing them helps readability.